tests/artist-read.test.js

```diff
@@ -0,0 +1,57 @@
+const { expect } = require('chai')
+const request = require('supertest')
+const db = require('../src/db')
+const app = require('../src/app')
+
+describe('Read Artists', () => {
```

Comment:
As before, the indentation here needs fixing. Every closing bracket for each describe and test block should be at the same level as the opening bracket so its easier to read the contents and see where each of them stop and start at a glance